docker/databricks/Dockerfile

```diff
@@ -0,0 +1,191 @@
+               sysstat ssh && \
+    /var/lib/dpkg/info/ca-certificates-java.postinst configure && \
+    # Initialize the default environment that Spark and notebooks will use
+    virtualenv -p python3.8 --system-site-packages /databricks/python3 \
+        && /databricks/python3/bin/pip install --no-cache-dir --upgrade pip \
+        && /databricks/python3/bin/pip install --no-cache-dir -r /tmp/requirements.txt \
+        # Install Python libraries for Databricks environment
+        && /databricks/python3/bin/pip cache purge && \
+    mkdir -p /databricks/jars && \
+    wget https://github.com/tsl0922/ttyd/releases/download/1.6.3/ttyd.x86_64 && \
+        mkdir -p /databricks/driver/logs && \
+        mkdir -p /databricks/spark/scripts/ttyd/ && \
+        mkdir -p /etc/monit/conf.d/ && \
+        mv ttyd.x86_64 /databricks/spark/scripts/ttyd/ttyd && \
+        export TTYD_BIN_FILE=/databricks/spark/scripts/ttyd/ttyd && \
+    apt-get -y purge --autoremove software-properties-common cuda-cudart-dev-${CUDA_MAJOR} cuda-cupti-dev-${CUDA_MAJOR} \
+               cuda-driver-dev-${CUDA_MAJOR} cuda-nvcc-${CUDA_MAJOR} cuda-thrust-${CUDA_MAJOR} \
+               python3.8-dev libpq-dev libcairo2-dev build-essential unattended-upgrades cmake ccache openmpi-bin \
+               linux-headers-5.4.0-117 linux-headers-5.4.0-117-generic linux-headers-generic libopenmpi-dev unixodbc-dev \
+               virtualenv python3-virtualenv && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* && \
+    mkdir -p /mnt/driver-daemon && \
+    #############
+    # Disable NVIDIA repos to prevent accidental upgrades.
+    #############
+    ln -s /databricks/jars /mnt/driver-daemon/jars && \
+    cd /etc/apt/sources.list.d && \
+    mv cuda.list cuda.list.disabled && \
+    # Create user "ubuntu"
+    useradd --create-home --shell /bin/bash --groups sudo ubuntu
+
+#############
+# Set up webterminal ssh
+#############
+ENV TTYD_DIR=/databricks/spark/scripts/ttyd
+ENV TTYD_BIN_FILE=$TTYD_DIR/ttyd
+   
+COPY webterminal/setup_ttyd_daemon.sh $TTYD_DIR/setup_ttyd_daemon.sh
+COPY webterminal/stop_ttyd_daemon.sh $TTYD_DIR/stop_ttyd_daemon.sh
+COPY webterminal/start_ttyd_daemon.sh $TTYD_DIR/start_ttyd_daemon.sh
+COPY webterminal/webTerminalBashrc $TTYD_DIR/webTerminalBashrc
+COPY webterminal/ttyd-daemon-not-active /etc/monit/conf.d/ttyd-daemon-not-active
+
+#############
+# Set all env variables
+#############
+ENV PYSPARK_PYTHON=/databricks/python3/bin/python3
+ENV DATABRICKS_RUNTIME_VERSION=10.4
+ENV LANG=C.UTF-8
+ENV USER=ubuntu
+ENV PATH=/usr/local/nvidia/bin:/databricks/python3/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin
+
+#############
+# Spark RAPIDS configuration
+#############
+ARG DRIVER_CONF_FILE=00-custom-spark-driver-defaults.conf
+ARG JAR_URL=https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/22.08.0/rapids-4-spark_2.12-22.08.0.jar
```

Comment:
```suggestion
ARG JAR_URL=https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/22.10.0/rapids-4-spark_2.12-22.10.0.jar
```

Reply:
could we also make it possible to point this to a local jar?